docs/src/main/sphinx/release/release-475.md

```diff
@@ -0,0 +1,61 @@
+# Release 475 (16 Apr 2025)
+
+## General
+
+* Publish additional metrics for input tables in event listener. ({issue}`25475`)
+* Expose dynamic filter statistics in the `QueryCompletedEvent`. ({issue}`25575`)
+* Improve scalability of inline data encoding in the parallel client protocol. ({issue}`25439`)
+* Improve performance of queries involving the `exclude_columns` table function. ({issue}`25117`)
+* Disallow dropping they `system` catalog. ({issue}`24745`)
+* Fix occasional query failures when adaptive planning is enabled. ({issue}`25411`)
+* Fix incorrect results when using window functions with `DISTINCT`. ({issue}`25434`)
+* Fix query failures with `EXCEEDED_LOCAL_MEMORY_LIMIT` errors due to incorrect memory accounting. ({issue}`25600`)
+* Properly handle inline session properties for `EXPLAIN` queries. ({issue}`25496`)
+
+## Security
+
+* Fix impersonation access control when access is granted via the role. ({issue}`25166`)
```

Comment:
@kokosing @homar, what was the failure?

Reply:
It was simply an Access Deny when it should not be. Details: https://github.com/trinodb/trino/issues/25166

Reply:
Fix incorrect access denial for access control with impersonation.